src/app/course-page/course-page.component.ts

```diff
@@ -1,15 +1,51 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { Courses } from '../models/course.interface';
```

Comment:
use model/models instead of interfaces. `course.interface -> course.model` or `course.models` if file gonna contains multiple interfaces
it's in angular style guide
![image](https://user-images.githubusercontent.com/26470573/99280247-cfd4aa80-2839-11eb-84b9-77cf08cb90dc.png)

Reply:
done